pkg/envoy/server.go

```diff
@@ -36,18 +38,31 @@ import (
 	"github.com/cilium/cilium/pkg/policy"
 	"github.com/cilium/cilium/pkg/policy/api"
 
-	"fmt"
 	"github.com/gogo/protobuf/sortkeys"
 	"github.com/golang/protobuf/proto"
 	"github.com/golang/protobuf/ptypes/duration"
 	"github.com/golang/protobuf/ptypes/struct"
 	"github.com/golang/protobuf/ptypes/wrappers"
 )
 
-// allowAction is a "Pass" route action to use in route rules. Immutable.
-var envoyRouteAllowAction = &envoy_api_v2_route.Route_Route{Route: &envoy_api_v2_route.RouteAction{
-	ClusterSpecifier: &envoy_api_v2_route.RouteAction_Cluster{Cluster: "cluster1"},
-}}
+var (
+	// allowAction is a "Pass" route action to use in route rules. Immutable.
+	envoyRouteAllowAction = &envoy_api_v2_route.Route_Route{Route: &envoy_api_v2_route.RouteAction{
+		ClusterSpecifier: &envoy_api_v2_route.RouteAction_Cluster{Cluster: "cluster1"},
+	}}
+
+	// xds is the xDS GRPC server.
+	xdsServer *XDSServer
+)
+
+// StartProxySupportServers starts the xDS GRPC server and the access log
+// server used by proxies.
+func StartProxySupportServers(stateDir string) {
```

Comment:
This function seems unused. Can we remove it?

Reply:
That was a mistake, it should have been used. Fixed.